constantine/eth_verkle_ipa/ipa_verifier.nim

```diff
@@ -111,18 +109,13 @@ func checkIPAProof* (ic: IPASettings, transcript: var CryptoHash, got: var EC_P,
 
   var g0 {.noInit.}: EC_P
 
-  var foldingScalars_big {.noInit.} : array[g.len,matchingOrderBigInt(Banderwagon)]
+  var foldingScalars_big {.noInit.} : array[EthVerkleDomain, matchingOrderBigInt(Banderwagon)]
 
-  for i in 0 ..< VerkleDomain:
+  for i in 0 ..< EthVerkleDomain:
     foldingScalars_big[i] = foldingScalars[i].toBig()
 
-  var g_aff {.noInit.} : array[VerkleDomain, EC_P_Aff]
-
-  for i in 0 ..< VerkleDomain:
-    g_aff[i].affine(g[i])
```

Comment:
2. Then g is converted to affine (but that could be done directly with SRS without copy)